apps/rit-tests-v9/src/react-19/components/Tooltip.cy.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
@mainframev just to confirm our offline convo, this is the correct approach in terms where the test should live. ty